src/main/kotlin/io/provenance/client/wallet/WalletSigner.kt

```diff
@@ -23,11 +23,14 @@ enum class NetworkType(
     MAINNET("pb", "m/505'/1'/0'/0/0")
 }
 
-class WalletSigner(networkType: NetworkType, mnemonic: String, passphrase: String = "") : Signer {
+class WalletSigner(prefix: String, path: String, mnemonic: String, passphrase: String = "") : Signer {
```

Comment:
I also have it on my `todo` list to be able to load "wallet/keys" with the ... uh shoot, i forget what it's called... @mtps told me the word. it's the thing that allows you load the keys at the path for your account, rather than requiring you to input the root mnemonic, which is riskier because it exposes all the keys in the entire hd wallet.

Reply:
@vwagner Are you thinking of extended keys--child keys derived from a root key?

Reply:
bip32?

Reply:
Yea, it's the bip32 encoded extended key. (tprv / xprv)